dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Planner/OpenAIPlanner.cs

```diff
@@ -136,12 +138,10 @@ public virtual async Task<string> CompletePromptAsync(ITurnContext turnContext,
 
                 return result;
             }
-            catch (AIException ex) when (ex.ErrorCode == AIException.ErrorCodes.Throttling)
+            catch (SemanticKernel.Diagnostics.HttpOperationException ex) when (ex.StatusCode == (HttpStatusCode)429)
```

Comment:
Add `SemanticKernel.Diagnostics` to `using ...` statements? Or any namespace conflict?

Reply:
Yes, Teams.AI has `HttpOperationException` as well.

Reply:
I see. Make sense.